src/main/java/seedu/duke/food/Food.java

```diff
@@ -0,0 +1,43 @@
+package seedu.duke.food;
+
+public class Food {
+    private final String name;
+    private final int calorie;
+    private final int carbohydrate;
+    private final int protein;
+    private final int fats;
+
+    public Food(String name, int calorie, int carbohydrate, int protein, int fats) {
+        this.name = name;
+        this.calorie = calorie;
+        this.carbohydrate = carbohydrate;
+        this.protein = protein;
+        this.fats = fats;
```

Comment:
Only fats is plural here.
Consider making all related names plural (might be more suggestive of type) or all singular.